packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -6269,5 +6269,97 @@ void main() {
     final Text hintTextWidget = tester.widget(hintTextFinder);
     expect(hintTextWidget.style!.overflow, decoration.hintStyle!.overflow);
   });
+
+  testWidgets('InputDecorator hint style has m3 defaults when enabled', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: true);
+    const String hintText = 'This is a hint.';
+    await tester.pumpWidget(
+       MaterialApp(
+        theme: theme,
+        home: const Material(
+          child: TextField(
+            decoration: InputDecoration(
+              hintText: hintText,
+              filled: true,
```

Comment:
Just wonder if we also need to check the hint style for the outlined text field? Because I saw both types are using same hint style.

Reply:
Looking at the other hint tests in this file, looks like none of them set the `filled` property so should be the default (false). I think if they use the same `hintStyle` we should be alright since this is just checking that.

Reply:
I see. Thx!